### ISSUE TITLE
PR appears as part of "unreleased

### ISSUE THREAD

**Issue by abonas** [github-changelog-generator/github-changelog-generator#282]
Generating the changelog on latest master of this repo:
https://github.com/abonas/kubeclient/

Shows that the PR https://github.com/abonas/kubeclient/pull/118
is part of "unreleased" section, however it was released as part of v 0.5.1
as can be seen here:
https://github.com/abonas/kubeclient/compare/v0.5.0...v0.5.1

**Comment by abonas** [comment]
Generating the changelog on latest master of this repo:
https://github.com/abonas/kubeclient/

Shows that the PR https://github.com/abonas/kubeclient/pull/118
is part of "unreleased" section, however it was released as part of v 0.5.1
as can be seen here:
https://github.com/abonas/kubeclient/compare/v0.5.0...v0.5.1

**Comment by skywinder** [comment]
@abonas  Thanks. I will take a look next week, when come from vacation.

**Comment by olleolleolle** [comment]
Hi, this is still a defect. I label this issue BUG now.

The tool puts PRs in the wrong bucket. 

In this small snippet, we can find PR 118 wrongly placed in the section generated after 0.5.1:

```
## [v0.6.0](https://github.com/abonas/kubeclient/tree/v0.6.0) (2015-09-21)
[Full Changelog](https://github.com/abonas/kubeclient/compare/v0.5.1...v0.6.0)

**Closed issues:**

- Parse Kubernetes config file to create clients [\#122](https://github.com/abonas/kubeclient/issues/122)
- Move kubeclient default version to kubernetes v1 [\#101](https://github.com/abonas/kubeclient/issues/101)

**Merged pull requests:**

- update default version of API to v1 instead of v1beta3 \[\#101\] [\#120](https://github.com/abonas/kubeclient/pull/120) ([jonmoter](https://github.com/jonmoter))
- check for missing item list returned due to k8b bug\[1\] [\#118](https://github.com/abonas/kubeclient/pull/118) ([moolitayer](https://github.com/moolitayer))
- move code that can be shared between clients to common [\#114](https://github.com/abonas/kubeclient/pull/114) ([moolitayer](https://github.com/moolitayer))
- Add k8s guestbook tests and get\_entities by namespace [\#105](https://github.com/abonas/kubeclient/pull/105) ([abonas](https://github.com/abonas))

## [v0.5.1](https://github.com/abonas/kubeclient/tree/v0.5.1) (2015-08-27)
[Full Changelog](https://github.com/abonas/kubeclient/compare/v0.5.0...v0.5.1)
```

(You can verify that the placement is wrong by visiting the 0.5.1 Full Changelog, where the 118 merge commit is visible.)

**Comment by mvz** [comment]
I think this problem occurs because the commit that is tagged with v0.5.1 has an author and commit date of 27 August, while the pull request was merged on 1 September. You can partly see this in the linked compare view https://github.com/ManageIQ/kubeclient/compare/v0.5.0...v0.5.1, and I have confirmed locally that both dates are, well, slightly off.

**Comment by mvz** [comment]
Checking now if this was fixed in #422.

**Comment by mvz** [comment]
I just tested with current `github_changelog_generator` and this bug no longer occurs. This is the part of the result for release 0.5.1:

```
## [v0.5.1](https://github.com/ManageIQ/kubeclient/tree/v0.5.1) (2015-08-27)

[Full Changelog](https://github.com/ManageIQ/kubeclient/compare/v0.5.0...v0.5.1)

**Merged pull requests:**

- check for missing item list returned due to k8b bug\[1\] [\#118](https://github.com/ManageIQ/kubeclient/pull/118) ([moolitayer](https://github.com/moolitayer))

## [v0.5.0](https://github.com/ManageIQ/kubeclient/tree/v0.5.0) (2015-08-27)
```
